src/Subscriber/AesSubscriber.php

```diff
@@ -141,8 +145,9 @@ private function encrypt(LifecycleEventArgs $args)
      * @param object $entity
      *
      * @return \ReflectionProperty[]
+     * @throws \ReflectionException
      */
-    private function getAnnotatedProperties($entity)
+    private function getAnnotatedProperties(object $entity): array
```

Comment:
i'm not sure `object` is not valid data type in PHP, can you please point me to this documentation? 🤔

Reply:
http://php.net/manual/en/migration72.new-features.php it's new in php 7.2